README.md

```diff
@@ -63,13 +63,20 @@ default_cluster: "YOUR_CLUSTER"
 ```
 
 ## Scheduling a job
-You are now ready to schedule jobs!
+You are now ready to schedule jobs! To run the following examples, make sure to install all dependencies:
+
+```bash
+pip install -e ".[extra]"
+```
+
 Let us have a look at `launch_hellocluster.py`, in particular, you can call the following to schedule a job:
 
 ```python
-cluster, partition = default_cluster_and_partition()
+user_path = Path('slurmpilot/config')
+config = load_config(user_path=user_path)
+cluster, partition = default_cluster_and_partition(user_path=user_path)
 jobname = unify("hello-cluster", method="coolname")  # make the jobname unique by appending a coolname
-slurm = SlurmWrapper()
+slurm = SlurmWrapper(config=config, clusters=[cluster])
```

Comment:
👍 thanks, it makes sense to have this one as this avoids creating an ssh connection to other clusters.

Reply:
actually it failed if I don't pass these parameters